docs/data/data-grid/list-view/ListViewAdvanced.js

```diff
@@ -25,15 +25,14 @@ import { formatDate, formatSize, stringAvatar } from './utils';
 import { ActionDrawer } from './components/ActionDrawer';
 import { RenameDialog } from './components/RenameDialog';
 
-export default function ListViewAdvanced(props) {
+export default function ListViewAdvanced({ window }) {
   // This is used only for the example - renders the drawer inside the container
-  const containerRef = React.useRef(null);
-  const container = () => containerRef.current;
```

Comment:
This was returning `null` when the drawer opened, meaning it did not have a container to render to.